src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -24,35 +24,33 @@
 import java.util.stream.IntStream;
 
 import emissary.test.core.UnitTest;
+import org.apache.commons.lang3.SystemUtils;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class ExecutrixTest extends UnitTest {
     private Executrix e;
-    private final boolean isWindows = System.getProperty("os.name").contains("Window");
+    private final boolean isWindows = SystemUtils.OS_NAME.contains("Windows");
 
     @Override
     @BeforeEach
     public void setUp() throws Exception {
         this.e = new Executrix();
 
-        final String tmp = System.getProperty("java.io.tmpdir");
-        if (tmp == null || (tmp.contains("~") && this.isWindows)) {
-            if (this.isWindows) {
-                File f = new File("c:/tmp");
+        final String tmp = TMPDIR;
+        System.out.println(tmp);
+        if ((tmp.contains("~") && this.isWindows)) {
```

Comment:
You probably can just remove this whole code block now. Since junit is setting the temporary dir, it is unlikely that we'll get a ~ in a windows path.